screens/PreviewDirections.js

```diff
@@ -316,14 +373,14 @@ export const styles = StyleSheet.create({
     DirectionTextHeader: {
         color: "#FFFFFF",
         fontSize: 25,
-        fontWeight: "bold",
         fontFamily: "encodeSansExpanded",
     },
     directionLabels: {
         fontWeight: "normal",
-        fontSize: 14,
+        fontSize: 20,
         color: "white",
-        width: "70%"
+        fontWeight: "bold",
+        width: "70%",
```

Comment:
There is a duplication here, fontWeight is used twice, once as normal and once as bold, please pick one.